### ISSUE TITLE
Project Meeting 03: Use Case

### ISSUE THREAD

**Issue by Valdarie** [Valdarie/INF2001-P3-T4#20]
**Task Title ID**: [#20](https://github.com/Valdarie/INF2001-P3-T4/issues/20)

**What is the task about? Please describe in detail.**
This project meeting was held to finalise the use case diagram features as well as to write up and discuss the use case descriptions. 

**What are the goals of this task? Please describe in detail.**
The goal of this meeting is to complete the graphic of the use case diagram. Additionally, by referring to the completed use case diagram, we are to write up the use case descriptions for each use case.  

**Success or completion criteria. Please describe in detail.**
In order for the success of this meeting, we have to complete the use case diagram as well as the use case description which are the following sections in the report:
Section 3.3 Use Case Diagram
Section 5 Use Case Description (Appendix A)

**Allocated Time**: 4 hours

**Start date and end date of the task.**
**Start Date:** 13 September 2023
**End Date**:  19 September 2023

**Who is the owner? (Task's person-in-charge; task is to be created and maintained by owner)**
Ashlinder

**The status of the task. (Usually in the initial column "To-Do")**
In Progress

**What is the priority of this task?**
High

**Comment by Valdarie** [comment]
Taken by: @CharistaSC BRYON TIAN
Subject at bottom of pic: @Valdarie VALERIE
![image](https://github.com/Valdarie/INF2001-P3-T4/assets/31137223/68e3f16b-d018-4dbd-b234-a31ca0f3bba6)

**Comment by Ashlinder** [comment]
**Reason For Delay:**
We are still in the process of getting feedback for our use case diagram and also continuing to refine and improve our diagram as well as descriptions.

**Comment by Ashlinder** [comment]
**Describe the work done in detail.**

In this meeting, the actors and their respective use cases were identified according to our software.
Additionally, we designed a use case diagram to include the actors and their use cases. 
Furthermore, we created a table for each use case that includes a detailed description.

**Has this work been committed to the repo?**
No.

**How does the work affect "4. Success or completion criteria"?**
Upon completion of this task, we successfully filled up the following sections of the report:
Section 3.3 Use Case Diagram
Section 5 Use Case Description (Appendix A)
Also, the use case diagram and descriptions will help us in understanding the flow of various processes in the software. 

**Start date and end date of the work done.**
**Start Date:** 13 Sep 2023
**End Date:** 19 Sep 2023
